### DISCUSSION TITLE
How to download github desktop for chrome os?

### DISCUSSION THREAD

**Discussion by SheaMcCann** [community/community#84897] [Other Feature Feedback, Questions, & Ideas] [answered]
Question

### Body

Is there a way to download github desktop as a .deb file or convert it to such? I know it's possible but I'm not sure how to do so. 

Or would a software such as wine and/or linux work to run the .exe file on chrome?

**Comment by megamiii** [comment, marked as answer]
For GitHub Desktop on Chrome OS, you can find the necessary .deb files in the Shiftkey GitHub repository. Here's the link to it: [Shiftkey Desktop on GitHub](https://github.com/shiftkey/desktop).

Hope this helps!

**Comment by ANAMASGARD** [comment]
https://youtu.be/BixOqdl_IAc?feature=shared
You can watch this video may be this will help .
